packages/opentelemetry-plugin-document-load/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@opentelemetry/plugin-document-load",
+  "version": "0.1.0",
+  "description": "OpenTelemetry document-load automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test:browser": "nyc karma start --single-run",
+    "tdd": "karma start",
+    "clean": "rimraf build/*",
+    "check": "gts check",
+    "codecov:browser": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "watch": "tsc -w"
```

Comment:
Similar to https://github.com/open-telemetry/opentelemetry-js/pull/431, could you please add `prepare` script here?

Reply:
@mayurkale22 it looks like adding `prepare` script for `web` and `document-load` is causing the build to fail for `ci/circleci: node12-browsers` any idea what can be missing or what is wrong?
@bg451 ^^

Reply:
Strange, I did the same in https://github.com/open-telemetry/opentelemetry-js/pull/431/files#diff-4588a531a488d7b28bc65bc94c63aa4eR16 for `web` and build is passed.

Reply:
the only difference is the version, can this be the issue ?

Reply:
after merging with latest master works fine